lib/widgets/my_workouts/workout_cards.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:flutter/material.dart';
+import 'package:workoutpersonalizer_frontend/constants/styles.dart';
+import 'package:workoutpersonalizer_frontend/models/exercise.dart';
+import 'package:workoutpersonalizer_frontend/models/workout.dart';
+import 'package:workoutpersonalizer_frontend/widgets/my_workouts/workout_card.dart';
+
+class WorkoutCards extends StatefulWidget {
+  const WorkoutCards({Key? key}) : super(key: key);
+
+  @override
+  _WorkoutCardsState createState() => _WorkoutCardsState();
+}
+
+class _WorkoutCardsState extends State<WorkoutCards> {
+  final List<WorkoutCard> _workoutCards = <WorkoutCard>[];
+  final ScrollController _scrollController = ScrollController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(child: _buildWorkoutCards());
+  }
+
+  Widget _buildWorkoutCards() {
+    // TODO: Get actual workouts from backend (WP-55)
+    final workouts = [
+      Workout(
+          "TestId",
+          "b70820ae-d0a3-411b-9217-0bf2370e7139",
+          "15 Minutes to Intense Abs",
+          "https://teamworkoutplatform.s3.us-west-2.amazonaws.com/Exercise+1+Thumbnail.png",
+          [Exercise()],
+          Duration(seconds: 510),
```

Comment:
Do you think we can format it to match the designs? So X min instead of the time format we have rn

Reply:
yep, I can do that